src/reducer/authReducer.ts

```diff
@@ -0,0 +1,32 @@
+const CHECK_LOGIN_DETAILS = 'CHECK_LOGIN_DETAILS';
+const LOGOUT_USER = 'LOGOUT_USER';
+const LOGIN_ON_START = 'LOGIN_ON_START';
+const REGISTER_NEW_USER = 'REGISTER_NEW_USER';
+
+export const initialAuthState: InitialAuthState = {
+	isUserLoggedIn: false,
+	userId: '',
+};
+
+export function authReducer(state: InitialAuthState, action: AuthActionTypes) {
+	switch (action.type) {
+		case CHECK_LOGIN_DETAILS:
```

Comment:
SET_LOGIN_DETAILS?

Reply:
Will refactor all, once the Frontend is connected to the Backend.